packages/datawell/src/react/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { Heading } from '@pluralsight/ps-design-system-text'
+import {
+  names as themeNames,
+  useTheme
+} from '@pluralsight/ps-design-system-theme'
+import {
+  RefForwardingComponent,
+  ValueOf
+} from '@pluralsight/ps-design-system-util'
+import { css } from 'glamor'
+import React from 'react'
+
+import stylesheet from '../css'
+
+const styles = {
+  dataWell: (themeName: ValueOf<typeof themeNames>) =>
+    css(
+      stylesheet['.psds-datawell'],
+      stylesheet[`.psds-datawell.psds-theme--${themeName}`]
+    ),
+  label: (themeName: ValueOf<typeof themeNames>) =>
+    css(
+      stylesheet['.psds-datawell__label'],
+      stylesheet[`.psds-datawell__label.psds-theme--${themeName}`]
+    ),
+  data: (themeName: ValueOf<typeof themeNames>) =>
+    css(
+      stylesheet['.psds-datawell__data'],
+      stylesheet[`.psds-datawell__data.psds-theme--${themeName}`]
+    ),
+  subLabel: (themeName: ValueOf<typeof themeNames>) =>
+    css(
+      stylesheet['.psds-datawell__sublabel'],
+      stylesheet[`.psds-datawell__sublabel.psds-theme--${themeName}`]
+    )
+}
+
+interface DataWellProps extends React.HTMLAttributes<HTMLDivElement> {
+  children: React.ReactNode
+  label: string
+  subLabel?: React.ReactNode
+}
+
```

Comment:
```suggestion
interface DataWellProps extends React.HTMLAttributes<HTMLDivElement> {
  label: string
  subLabel?: React.ReactNode
}

```

Reply:
child node was marked as required on the old proptypes.  Given that, would you still make this change (because React.HTMLAttributes<HTMLDivElement> is going to make it optional)?

Reply:
hmmm.....i probably would make the change - but i get your point. i like not needing to add props that are already covered. i leave it up to you to make an exec decision based on component usage